.circleci/main.yml

```diff
@@ -1163,7 +1163,60 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/mount-server/mount-server_${CIRCLE_TAG:1}* --generate-notes
             fi
+  console-release-draft:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - gh/setup:
+          version: 2.13.0
+      - run:
+          name: create github console release draft
+          command: |
+            if [[ $CIRCLE_TAG == *"-"* ]];
+              then
+                  echo "git tag has - assuming its a prerelease."
+                  parentBranch=master
+              else
+                  echo "regular release."
+                  parentBranch=${CIRCLE_TAG:1}
+              fi
+            git clone https://github.com/pachyderm/haberdashery.git haberdashery
+            git checkout -b "release-${CIRCLE_TAG:1}"
+            tmp=$(mktemp)
+            jq --arg val "${CIRCLE_TAG:1}-${CIRCLE_SHA1}" '.pachyderm = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_SHA1}" '.pachReleaseCommit = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_TAG:1}" '.pachctl = $val' version.json > "$tmp" && mv "$tmp" version.json
+            git commit -m "bump pachyderm version"
```

Comment:
I think there needs to be a `git add` somewhere between the `clone` and `commit`? Are we in the haberdashery folder/repo as well, or are we supposed to be in the `pachyderm` repo still?

Reply:
in the haberdashery repo, good catch. I was able to test most of this. unfortunately not really all testable until a release is done. but made it to where we can still intervene manually since no other job depends on it.

Reply:
Yeah, I figured this is one of those things that's hard to test until you use it. I think it still needs a `cd haberdashery` somewhere though

Reply:
@djanicekpach I think you have a good point. I think, maybe I should wait to merge this, run these steps on the next release as a test and then merge it once I confirm they all work. 

I believe the PR will never close as their might not be an approver. So might need to add that step too.

Reply:
@molinamelendezj did the steps work for the last release?